### ISSUE TITLE
Surface Pro 11 with Apple Trackpad 2

### ISSUE THREAD

**Issue by shaps80** [imbushuo/mac-precision-touchpad#540]
I note the last release was a few years back. If I compile from source can I get this to work on arm with latest snapdragon surface pro or will this not work?

Thanks

**Comment by AaronMFJ** [comment]
Interested in this as well.

**Comment by dbhope64** [comment]
Yes, I have a Surface Pro 11th edition.  I want to use the Apple Magic Trackpad on it.  I have been trying all the suggestions I can find for days now. Has anyone been successful?
